ui/helper.c

```diff
@@ -537,14 +537,17 @@ void UI_DrawRxDBM(uint16_t RXdBM, bool isNeg, uint16_t len, uint8_t Vfo, bool Cl
 	if (Clear) {
 		UI_DrawSmallString(105, Y, "    ", 4);
 	} else {
+		for (int i=1; i<3; i++) {
```

Comment:
```suggestion
		for (int i = len; i < 3; i++) {
```
Let's save a few CPU cycles.